Build/scripts/compose_build.ps1

```diff
@@ -14,15 +14,6 @@ param (
 )
 
 #
-# Clean up the sentinel which previously marked this build as incomplete.
-#
-
-$buildIncompleteFile = Join-Path $rootPath "build.incomplete"
```

Comment:
Right. `build.incomplete` didn't provide any new information but there's sometimes contention for locks on the file from the different parallel builds, so having it is a net loss. It's also another moving piece to deal with for the rebuild scenario.